server/src/main/java/org/opensearch/node/Node.java

```diff
@@ -454,6 +456,15 @@ protected Node(
             // Ensure to initialize Feature Flags via the settings from opensearch.yml
             FeatureFlags.initializeFeatureFlags(settings);
 
+            final List<IdentityPlugin> identityPlugins = new ArrayList<>();
+            if (FeatureFlags.isEnabled(FeatureFlags.IDENTITY)) {
+                // If identity is enabled load plugins implementing the extension point
+                logger.info("Identity on so found plugins implementing: " + pluginsService.filterPlugins(IdentityPlugin.class).toString());
+                identityPlugins.addAll(pluginsService.filterPlugins(IdentityPlugin.class));
+            }
+            logger.info("Creating identity service in Node.java with identity plugins: " + identityPlugins.toString());
```

Comment:
Oops, @scrawfor99 my apologies, I missed that, probably `Node.java` is too precise:

```suggestion
            logger.info("Creating identity service for node with identity plugins: " + identityPlugins.toString());
```

Reply:
Oops, I did not mean to include that print. I will remove it.